thavalon-webapp/src/components/gameComponents/roleInformation.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import { GameSocket, InboundMessage, InboundMessageType, OutboundMessageType } from "../../utils/GameSocket";
+import "../../styles/gameStyles/roleInformation.scss";
+
+interface RoleInfo {
+    abilities: string,
+    assassinatable: boolean,
+    description: string,
+    isAssassin: boolean,
+    otherInfo: string,
+    priorityTarget: string | undefined,
+    role: string,
+    seenPlayers: string[],
+    team: string,
+    teamMembers: string[],
+}
+
+interface Snapshot {
+    roleInfo: RoleInfo,
+    missions: string[],
+    log: string[],
+}
+
+export function RoleInformation(): JSX.Element {
+    const [roleInfoElement, setRoleInfoElement] = useState<JSX.Element>(<h1>Loading Role Info...</h1>);
+
+    /**
+     * Creates the JSX Element for representing the role information
+     * section of a game page.
+     * @param roleInfo The role info received in a snapshot message.
+     */
+    function createRoleInfoElement(roleInfo: RoleInfo): JSX.Element {
+        return (
+            <div id="playerInfo">
+                <h1 className="gameSectionHeader">Player Info</h1>
```

Comment:
Should we be consistent with CSS class names? I think hyphenated names are usually the convention for CSS classes.